File: 215_Kth_Largest_Element_in_an_Array.py
def findKthLargest(nums, k):
    nums.sort()
    return nums[len(nums) - k]

if __name__ == '__main__':
    print(findKthLargest(nums = [3,2,1,5,6,4], k = 2))
    print(findKthLargest(nums = [3,2,3,1,2,4,5,5,6], k = 4))

File: 215_Kth_Largest_Element_in_an_Array.cpp
#include <iostream>
#include <vector> 

using namespace std;

class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        // int output = nums[k];
        sort(nums.begin(), nums.end());
        cout << "Sorted \n";
        for (auto x : nums)
            cout << x << " ";

        int index = nums.size() - k;
        int output = nums[index];
        return output;
    }
};

// void printVector(vector<int> & vec) {
//         for (int i: vec) {
//             cout << i << ' ';
//         }
//     cout << endl;
// }

int main(){
    Solution s;
    vector<int> nums = {3,2,1,5,6,4};
    int ans = s.findKthLargest(nums, 2);
    cout << ans << endl;
    return 0; 
}